sites/docker-compose.berwyn.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+    hub-be:
+        environment:
+            DRUPAL_APP_URI: https://drupal.digital-hub-stage.hmpps.dsd.io
+            HUB_DB_ENV_MYSQL_DATABASE: hubdb
+            HUB_DB_ENV_MYSQL_USER: hubdb_user_2y478gt
+            HUB_DB_PORT_3306_TCP_ADDR: hub-db
+            HUB_EXT_FILE_URL: http://digital-hub.bwi.dpn.gov.uk:11001/sites/default/files
+            PHP_MEMORY_LIMIT: 256M
+            PHP_UPLOAD_MAX_FILE_SIZE: 1024M
+            PHP_POST_MAX_SIZE: 1024M
+            PIWIK_URI: //digital-hub.bwi.dpn.gov.uk:12001
+
+
+        volumes:
+            - /content/moj_dhub_prod001_app/usr/share/nginx/html/moj_be/sites/default/files:/var/www/html/sites/default/files
+
+    hub-db:
+        volumes:
+            - /data/hub-db:/var/lib/mysql:rw
+
+    hub-matomo:
+        volumes:
+            - /data/moj_dhub_matomo_config:/var/www/html/config:rw
+
```

Comment:
hub-matomo-db will need the data volumes adding

/data/moj_dhub_matomo_db/var/lib/mysql/:/var/lib/mysql/

Reply:
Same for Wayland, and maybe staging?